pkg/grpc/databroker/databroker.go

```diff
@@ -8,6 +8,16 @@ import (
 	"strings"
 )
 
+// TypeURLs for Any.
+const (
+	ConfigTypeURL         = "type.googleapis.com/pomerium.config.Config"
+	DirectoryGroupTypeURL = "type.googleapis.com/directory.Group"
+	DirectoryUserTypeURL  = "type.googleapis.com/directory.User"
+	SessionTypeURL        = "type.googleapis.com/session.Session"
+	ServiceAccountTypeURL = "type.googleapis.com/user.ServiceAccount"
+	UserTypeURL           = "type.googleapis.com/user.User"
+)
```

Comment:
if we're moving that into grpc package, we can make them all enums

Reply:
sorry I don't understand. Go doesn't have enums. What do you want me to change?

Reply:
I mean as long as all types are moved to `pkg/grpc/databroker`, we can also declare all types explicitly in `.proto` via enums and have the record types explicitly typed. 

or if we leave them just in Go code, then we can still keep them strongly typed like that 

```golang
type RecordType string

const (
	ConfigTypeURL         = RecordType("type.googleapis.com/pomerium.config.Config")
	DirectoryGroupTypeURL = RecordType("type.googleapis.com/directory.Group")
	DirectoryUserTypeURL  = RecordType("type.googleapis.com/directory.User")
	SessionTypeURL        = RecordType("type.googleapis.com/session.Session")
	ServiceAccountTypeURL = RecordType("type.googleapis.com/user.ServiceAccount")
	UserTypeURL           = RecordType("type.googleapis.com/user.User")
)
```

Reply:
I see. These type URLs are for the any type. I can update the code to generate them at runtime.

Reply:
Oh nevermind, that will introduce a cyclic dependency. I can make a method in grpctuil I guess.